languages/python/omni_http/src/omni_http/omni_httpd/__init__.py

```diff
@@ -15,11 +16,11 @@ class HTTPRequest:
     headers: list[HttpHeader]
 
     def wsgi_environ(self) -> dict:
-        environ = {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': self.body or b"",
-                   'wsgi.multithread': False,
+        environ = {'wsgi.version': (1, 0), 'wsgi.url_scheme': 'http', 'wsgi.input': io.BytesIO(self.body or b""),
+                   'wsgi.input_terminated': 1, 'wsgi.multithread': False,
```

Comment:
It looks like https://peps.python.org/pep-3333/ doesn't define input_terminated. I wonder what the authoritative reference for this is?

Reply:
It is defined in a flask extension: https://tedboy.github.io/flask/_modules/werkzeug/wsgi.html

Alternatively I think we can also set the content-length header, but either way the logic for retrieving the stream is defined and used from this extension.

Reply:
Interesting. Trying to find some more information if this is Flask-specific and if anything else uses this.

Reply:
this has been changed to using `content-length` instead of `input_terminated`